tests/tests_pytorch/deprecated_api/test_remove_1-8.py

```diff
@@ -1132,17 +1132,37 @@ def test_trainer_gpus(monkeypatch, trainer_kwargs):
     monkeypatch.setattr(torch.cuda, "device_count", lambda: 4)
     trainer = Trainer(**trainer_kwargs)
     with pytest.deprecated_call(
-        match="`Trainer.gpus` was deprecated in v1.6 and will be removed in v1.8."
-        " Please use `Trainer.num_devices` or `Trainer.device_ids` to get device information instead."
+        match=(
+            "`Trainer.gpus` was deprecated in v1.6 and will be removed in v1.8."
+            " Please use `Trainer.num_devices` or `Trainer.device_ids` to get device information instead."
+        )
     ):
         assert trainer.gpus == trainer_kwargs["devices"]
 
+    with pytest.deprecated_call(
+        match=(
+            "`AcceleratorConnector.gpus` was deprecated in v1.6 and will be removed in v1.8. Please use"
+            " `Trainer.num_devices` or `Trainer.device_ids` to get device information instead."
+        )
+    ):
+        assert trainer._accelerator_connector.gpus == trainer_kwargs["devices"]
```

Comment:
since accelerator_connector is protected.. should it rather be removed?

Reply:
I'd be a bit cautious about removing it completely. Sure, it's protected, so officially "no guarantees", but it's not like it's hard to find and I can easily imagine some user code using it

Reply:
we provide separate trainer properties for them.

Reply:
Fine then let's just remove them